contents/blog/helping-engineers-to-product.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2023-02-16
+title: "CEO diary: Helping engineers to be better at product"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: CEO diaries
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+One of the things I've learned at PostHog, is the simpler a strategy, the more likely it's the right one.
+
+We simplified [our strategy](../handbook/strategy/overview) recently. This post explains the context.
+
+## The change
+
+Our company mission has always been to increase the number of successful products in the world.
+
+The change is that we've realized that the simple way PostHog can achieve this is to help engineers be better at product.
+
+Today, the people writing the code for the products you use are so often treated as a resource. They don't get a say in what gets built, just picking up ticket after ticket. Yet, there's a reason they're engineers in the first place – they were a combination of intelligent and hard-working, which meant they built deep skills.
+
+Our competitors near universally focus on helping non-technical product managers, who write tickets for their developers.
+
+We think this is backwards. Look at the way most teams work on tickets today:
+
+* Endless alignment
+* Endless requirements gathering
+* Endless handoffs (design/product/engineering/sales/marketing/support)
+
+If we reduce the need for PMs and help engineers go further by themselves, we reduce time lost on the above activities. Moreover, the best engineers usually want to work for companies where they get more autonomy.
+
+## What's the strategy then?
+
+### Provide every tool needed for evaluating feature success
+
+The best use of an engineer's time is to ship features that have an impact on customers. Currently, this requires a large number of tools and product managers to pull all the insights together. By integrating all these tools we can make this easy – no integration needed, no extra vendors, no extra javascript, and workflows to guide engineers through feature development, from initial idea, to release, measurement, gathering qualitative data, and back to the start.
+
+### Get in first
+
+By already being used by our customers, we’re the default for each additional tool they add. It’s the technical co-founder and early engineers building the MVP and integrating the first product tools, not PMs. By focusing on engineers we can be their preferred choice and get in first. Additionally, we can ladder our tools – for example, session recording is used much earlier in the life cycle of the product than others like the CDPs helping us get in earlier than competing products. As a result, we aren't heavily focused on enterprise – we even [sunsetted K8s support](sunsetting-helm-support-posthog) as part of this change.
```

Comment:
```suggestion
It’s the technical co-founder and early engineers building the MVP and integrating the first product tools, not PMs. By focusing on engineers we can get in first, and later become the default choice for each additional tool they add. 

Additionally, we can ladder our tools – session recording is used much earlier in the life cycle of the product than others, like the customer data platforms (CDP), helping us get in earlier than competing products. As a result, we aren't heavily focused on enterprise – we even [sunsetted K8s support](sunsetting-helm-support-posthog) as part of this change.
```

One small thought. In our announcement we mainly framed the sunsetting as related to support needs, but this (slightly) contradicts that. Does this matter? It's probably ok imo, more of an FYI.